src/pke/lib/scheme/ckksrns/ckksrns-fhe.cpp

```diff
@@ -264,7 +284,7 @@ Ciphertext<DCRTPoly> FHECKKSRNS::EvalBootstrap(ConstCiphertext<DCRTPoly> ciphert
     double powP  = pow(2, p);
 
     double deg        = std::round(std::log2(qDouble / powP));
-    double correction = 9.0 - deg;
+    double correction = m_correctionFactor - deg;
```

Comment:
Just to confirm. I don't believe we should be using the correctfactor for NATIVE_SIZE=128. Is this what we have in the implementation?